internal/lib/container_server_test.go

```diff
@@ -72,24 +72,6 @@ var _ = t.Describe("ContainerServer", func() {
 			Expect(err).NotTo(BeNil())
 			Expect(server).To(BeNil())
 		})
-
-		It("should fail with invalid hooks dir", func() {
```

Comment:
we seem to have lost this?

Reply:
Yes, the libpod bump made this code path not possible any more.